pkg/kubectl/resource_printer.go

```diff
@@ -590,6 +592,40 @@ func printReplicationControllerList(list *api.ReplicationControllerList, w io.Wr
 	return nil
 }
 
+func printJob(job *expapi.Job, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
+	containers := job.Spec.Template.Spec.Containers
+	var firstContainer api.Container
+	if len(containers) > 0 {
+		firstContainer, containers = containers[0], containers[1:]
+	}
+	_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%d\n",
+		job.Name,
+		firstContainer.Name,
+		firstContainer.Image,
+		labels.FormatLabels(job.Spec.Selector),
+		job.Status.Successful)
+	if err != nil {
+		return err
+	}
+	// Lay out all the other containers on separate lines.
+	for _, container := range containers {
+		_, err := fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\n", "", container.Name, container.Image, "", "")
```

Comment:
Please read:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md

No multi-line get output. Could do that in describe, which you should implement.

I'd push the kubectl changes to another PR. This one is pretty big.